runner/app.go

```diff
@@ -77,15 +91,27 @@ func (a *App) Start(ctx context.Context) error {
 		return nil
 	})
 
-	log.Infof("Starting runner....")
+	log.Infof("Building catalog....")
 	g.Go(func() error {
-		err := a.runner.Start(ctx)
+		err := a.deps.runnerService.BuildCatalog()
 		if err != nil {
 			return err
 		}
 		return nil
 	})
 
+	// Commenting the Start function as it collides with the catalog creation
+	/*
+		log.Infof("Starting runner....")
+		g.Go(func() error {
+			err := a.deps.runnerService.Start(ctx)
+			if err != nil {
+				return err
+			}
+			return nil
+		})
+	*/
+
```

Comment:
Can we delete this?

Reply:
Yes, of course

Reply:
Done: https://github.com/trento-project/runner/pull/3/commits/cbcd45d0550fc2997262ceda1a8005a35ccabfca